packages/venia-concept/src/components/Header/header.css

```diff
@@ -56,3 +56,60 @@
     grid-template-columns: 1fr 1fr;
     justify-items: end;
 }
+
+.searchFallback {
+    composes: form from '../SearchBar/searchBar.css';
+    padding: 0 1rem;
+}
+
+.input {
+    composes: input from '../TextInput/textInput.css';
+}
+
+.loader,
+.loader:before,
+.loader:after {
+    border-radius: 50%;
+    width: 2em;
+    height: 2em;
+    animation-fill-mode: both;
+    animation: pulse 1.8s infinite ease-in-out;
+}
+
+.loader {
+    color: rgb(var(--venia-grey-dark));
+    font-size: 6px;
+    margin: calc(-2em + 6px) auto 0;
```

Comment:
Why the calc?

Reply:
The animation is using box shadow, which needed to be offset by the height and font-size. Pulled this from an animation tutorial, so happy to use another effect if you have a suggestion.

Reply:
Oh I see. Maybe pull those into a `var` each to be clear about whats going on?